boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig

```diff
@@ -14,3 +14,11 @@ CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+
+CONFIG_MULTI_LEVEL_INTERRUPTS=y
+CONFIG_2ND_LEVEL_INTERRUPTS=y
+CONFIG_MAX_IRQ_PER_AGGREGATOR=32
```

Comment:
These all seem like they should be done in the SoC defconfig as well; am I missing something?